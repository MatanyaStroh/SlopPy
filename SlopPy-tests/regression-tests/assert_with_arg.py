# test out an assertion with an argument

def foo():
  x = 5
  assert x > 10, x

foo()
