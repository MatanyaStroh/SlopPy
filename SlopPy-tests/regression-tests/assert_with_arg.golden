{NA <type 'exceptions.AssertionError'> AssertionError(5,)
 Traceback (most recent call last):
  File "regression-tests/assert_with_arg.py", line 5, in foo
    assert x > 10, x
}
{NA <type 'exceptions.AssertionError'> AssertionError(5,)
 Traceback (most recent call last):
  File "regression-tests/assert_with_arg.py", line 5, in foo
    assert x > 10, x
}
{NA <type 'exceptions.AssertionError'> AssertionError(5,)
 Traceback (most recent call last):
  File "regression-tests/assert_with_arg.py", line 5, in foo
    assert x > 10, x
}
